### PR TITLE
[SRVCLI-250] Add test-images and test-e2e-local cmds to Makefile

### PR DESCRIPTION
**Pull request by dsimansk**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,6 +38,12 @@ build-cross-package: build-cross
 test-install:
 	go install $(TEST_IMAGES)
 .PHONY: test-install
+	
+test-images:
+	for img in $(TEST_IMAGES); do \
+		KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ko resolve --tags=latest -RBf $$img ; \
+	done
+.PHONY: test-image
```
Comment (navidshaikh) [inline Makefile:46]:
```suggestion
.PHONY: test-images
```

File: Makefile [hunk 2]
```diff
@@ -47,6 +53,13 @@ test-e2e:
 	./openshift/e2e-tests-openshift.sh
 .PHONY: test-e2e
 
+# Run make DOCKER_REPO_OVERRIDE=<your_repo> test-e2e-local if test images are available
+# in the given repository. Make sure you first build and push them there by running 
+# `make DOCKER_REPO_OVERRIDE=<your_repo> test-images`.
+test-e2e-local:
+	KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ./test/local-e2e-tests.sh
```
Comment (navidshaikh) [inline Makefile:60]: We'd need anyuid hack for --user e2e tests

```
  # Add anyuid scc to all authenticated users so e2e tests for --user flag can user any user id
  oc adm policy add-scc-to-group anyuid system:authenticated
```

beside, I havent tried this locally lately (after we added the image building in CI)

`KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ./test/local-e2e-tests.sh`

Comment (dsimansk) [reply Makefile]: Those should be resolved in the latest commits.

**Comment by dsimansk** [comment]
Test run PR: https://github.com/openshift/knative-client/pull/501.

**Review by navidshaikh — APPROVED** [review]
/lgtm

/hold for @mvinkler to confirm if the changes worked fine for him.

**Review by mvinkler — APPROVED** [review]

**Comment by mvinkler** [comment]
@navidshaikh I am not listed in https://github.com/openshift/knative-client/blob/master/OWNERS_ALIASES so it's up to you to /lgtm and /unhold

**Review by navidshaikh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**